routing.rst

```diff
@@ -2763,6 +2763,11 @@ service, which you can inject in your services or controllers::
         }
     }
 
+.. versionadded:: 7.3
+
+    Signed URI hashes no longer include the ``/`` or ``+`` characters, as these
+    may cause issues with certain clients, since Symfony 7.3.
```

Comment:
This doesn't sound like something we have to have in the documentation (I'm aware you didn't add it in this PR).

Seems more like a bugfix worth mentioning in the changelog than something super important that we have to tell everyone reading the main routing guide 🙂

Reply:
ok, removed.